comms/examples/stress/service.rs

```diff
@@ -44,10 +44,17 @@ use tari_comms::{
 };
 use tari_crypto::tari_utilities::hex::Hex;
 use tokio::{task, task::JoinHandle};
+use tari_comms::rate_limit::RateLimit;
+use tokio::time::Duration;
+
+/// The minimum amount of inbound messages to accept within the `RATE_LIMIT_RESTOCK_INTERVAL` window
+const RATE_LIMIT_MIN_CAPACITY: usize = 1;
```

Comment:
This is not the minimum capacity - it's just the capacity
```suggestion
const RATE_LIMIT_CAPACITY: usize = 1;
```